functions/hold_peak.h

```diff
@@ -0,0 +1,59 @@
+#ifndef FUNCTIONS_HOLD_PEAK_H
+#define FUNCTIONS_HOLD_PEAK_H
+
+// Usage: HoldPeakF<F, hold_time_, SPEED>
+// Holds Peak value of F for hold_time_ in millis.
+// then transitions down over SPEED to current F
+// F, SPEED: FUNCTION
+// hold_time_ MILLIS
+// return value: FUNCTION, same for all LEDs
+
+class BladeBase;
+
+template<class F, int hold_time_, class SPEED>
+
+class HoldPeakF {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    current_ = f_.getInteger(0);
```

Comment:
current_ doesn't need to be a global variable.

Reply:
fixed